tests/test_ates.py

```diff
@@ -11,9 +11,18 @@
 class TestAtes(TestCase):
     def test_ates(self):
         """
-        This test checks the constraints concerning the heat2discharge and energy conservation for
-        the ates. THe heat loss model used annd the typical cyclic constraint that will be applied
-        in most use cases.
+        This test checks the constraints concerning the heat to discharge and energy conservation
+        for the ates. THe heat loss model used annd the typical cyclic constraint that will be
+        applied in most use cases.
+
+        Checks:
+        - The heat loss is computed as expected (loss coef * stored heat [J])
+        - Checks that the efficiency causes less energy to be discharged than charged
+        - Cyclic storage behaviour
+        - standard energy conservation, etc.
```

Comment:
Consistency- bullets sentences to start with capital letter or not?

Reply:
done